DiffusionPreprocessing/scripts/DiffusionToStructural.sh

```diff
@@ -97,31 +95,59 @@ done
 ${GlobalScripts}/Rotate_bvecs.sh "$DataDirectory"/bvecs "$WorkingDirectory"/diff2str.mat "$T1wOutputDirectory"/bvecs
 cp "$DataDirectory"/bvals "$T1wOutputDirectory"/bvals
 
+DilateDistance=`echo "$DiffRes * 4" | bc`  # Extrapolates the diffusion data up to 4 voxels outside of the FOV
+
 #Register diffusion data to T1w space. Account for gradient nonlinearities if requested
```

Comment:
Can we update this comment:
`#Register diffusion data to T1w space. Account for gradient nonlinearities (using one-step resampling) if requested`

Reply:
Done